my-app/src/App.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import logo from './logo.svg'
+import './App.css';
+
+class Board extends Component {
+  constructor() {
+    super();
+    this.size = 9;
+    this.state = {
+
+      // wIsNext: true,
+      board: this.createGameBoard(this.size),
+
+    };
+  }
+
+  createGameBoard(size){
+    let gameBoard = [];
+      for(let x = 0; x < size; x++){
```

Comment:
This indentation is confusing; it is best practice to consistently left align code in the same block.